helpers/lib_tasks_docker_release.py

```diff
@@ -508,28 +637,23 @@ def docker_release_multi_build_dev_image(  # type: ignore
     # is used.
     dev_version = _get_dev_version(version, container_dir_name)
     # 2) Run tests for the "local" image.
-    if skip_tests:
-        _LOG.warning("Skipping all tests and releasing")
-        fast_tests = False
-        slow_tests = False
-        superslow_tests = False
-        qa_tests = False
-    stage = "local"
-    if fast_tests:
-        hlitapyt.run_fast_tests(ctx, stage=stage, version=dev_version)
-    if slow_tests:
-        hlitapyt.run_slow_tests(ctx, stage=stage, version=dev_version)
-    if superslow_tests:
-        hlitapyt.run_superslow_tests(ctx, stage=stage, version=dev_version)
     # 3) Run QA tests using the local version of an image.
     # Use the local image because it is not possible to tag a multi-arch
     # image as dev without releasing (pushing) it.
     # The difference between a local and a dev image is just a tag.
-    if qa_tests:
-        hlitapyt.run_qa_tests(ctx, stage="local", version=dev_version)
+    _run_tests(
+        ctx,
+        "local",
```

Comment:
We need to understand, what the "local" is, so, let's assign it to variable before and then pass to the function.